samples/snippets/fsharp/VS_Snippets_CLR/GetCustomAttributes/FS/ca2.fs

```diff
@@ -0,0 +1,36 @@
+﻿module ca2
+
+// <Snippet2>
+open System
+open System.ComponentModel
+
+// Assign some attributes to the module.
+[<``module``: Description "A sample description">]
+
+// Set the module's CLSCompliant attribute to false
+// The CLSCompliant attribute is applicable for /target:module.
+[<``module``: CLSCompliant false>]
+do ()
+
+type DemoClass = class end
+
+// Get the Module type to access its metadata.
+let module' = typeof<DemoClass>.Module
```

Comment:
Same comment: the variable should not end in `'`:

```suggestion
let module = typeof<DemoClass>.Module
```

Reply:
module is a reserved keyword, I'll change it something else.